MANIFEST.in

```diff
@@ -1 +1,2 @@
 include alibi/data/cats.tar.gz
+include extra-requirements.txt
```

Comment:
Amend once file moved.

Reply:
I'm not sure we can do this and also keep the tox tests due to [this issue](https://stackoverflow.com/questions/57611665/tox-fails-because-setup-py-cant-find-the-requirements-txt). Either we keep this line in the `MANIFEST.in` file or I suppose we could store the extra requirements as a string/dict within the `setup.py` file?

Reply:
Not sure I follow, I meant to ammend the line to the new location of the file?

Reply:
ah, sorry i misunderstood